client/common/config/config.js

```diff
@@ -1,4 +0,0 @@
-modules.define('config', function (provide) {
```

Comment:
if so I'd suggest to use `page` DOM node (not to create an extra one) and same way with `data-bem` to provide data to client. and then expose it as a `config` module